protocol/messenger_contacts.go

```diff
@@ -45,157 +45,182 @@ func (m *Messenger) AcceptContactRequest(ctx context.Context, request *requests.
 	return response, nil
```

Comment:
Changes in this file are actually not too big as it might seem — better to open it all in editor to investigate.